src/commands/math/basicSymbols.ts

```diff
@@ -200,22 +227,22 @@ var Variable = P(Symbol, function(_, super_) {
     } else {
       if (this[L] && !(this[L] instanceof Variable)
           && !(this[L] instanceof BinaryOperator)
-          && this[L].ctrlSeq !== '\\ ')
+          && (this[L] as MQNode).ctrlSeq !== '\\ ')
```

Comment:
Minor note, even if we made the coalescing behavior `.ctrlSeq || ''` happen unconditionally when we access the property as I suggested above, it wouldn't break this usage that doesn't do it (because it obviously doesn't do anything useful here)